config/sync/jsonapi_extras.jsonapi_resource_config.block--block.yml

```diff
@@ -0,0 +1,51 @@
+uuid: 2763f554-cd5d-4c4e-a9df-4cd228f1a8b2
+langcode: en
+status: true
+dependencies: {  }
+id: block--block
+disabled: false
+path: blocks
+resourceType: blocks
+resourceFields:
+  id:
+    fieldName: id
+    publicName: internalId
+    enhancer:
+      id: ''
+    disabled: false
+  status:
+    fieldName: status
+    publicName: isPublished
```

Comment:
Booleans are a yes or no response to a question. Note the difference:

— What's the _status_ of this node?
— Yes.

vs

— Is this node published?
— Yes.